src/sendGCM.js

```diff
@@ -87,6 +87,7 @@ const sendGCM = (regIds, data, settings) => {
     body_loc_key: data.locKey, // Android, iOS
     body_loc_args: data.locArgs, // Android, iOS
     title_loc_key: data.titleLocKey, // Android, iOS
+    android_channel_id: data.android_channel_id, // Android
```

Comment:
could you move it beneath _title_loc_args_ in order to keep _title_loc_key_ and _title_loc_args_ grouped together?

Reply:
could you add a test for the new payload key, to ensure the functionality won't get removed by accident?

Reply:
please add it to the example in the `README.md` as well